lib/hydra/derivatives/services/capability_service.rb

```diff
@@ -0,0 +1,17 @@
+require 'open3'
+
+module Hydra::Derivatives
+  class CapabilityService
```

Comment:
I'm finding a bit of a disconnect between the class name (`CapabilityService`) and what it is doing. I believe a strong version of this class would be one that I could do the following:

```ruby
class Hydra::Derivatives::AudioEncoder
  def initialize(ffmpeg_output: capture_ffmpeg_output)
    @ffmpeg_output = ffmpeg_output
  end
  # @return [String] name of the the shell command to perform audio encoding
  def installed_audio_encoder_name
    if fdk_aac?
      'libfdk_aac'
    else
      'aac'
    end
  end
end

Hydra::Derivatives::AudioEncoder.new.installed_audio_encoder_name
=> 'aac' # or 'libfdk_aac'
```